libnopegl/src/backends/vk/gpu_ctx_vk.c

```diff
@@ -1522,7 +1522,7 @@ const struct gpu_ctx_class ngli_gpu_ctx_vk = {
 
     .set_viewport                       = vk_set_viewport,
     .set_scissor                        = vk_set_scissor,
-                                                                                 
+
```

Comment:
another thing to squash somewhere I believe

Reply:
Done.